src/models/me/mod.rs

```diff
@@ -251,6 +252,17 @@ impl Me {
         self.post("api/editusertext", &form).await
     }
 
+    /// Get submissions by id
+    ///
+    /// # Arguments
+    ///
+    /// * `ids` - the fullnames of submisions to get, comma seperated
+    pub async fn get_submissions(&self, ids: &str) -> Result<Submissions, RouxError> {
```

Comment:
this should have the `#[maybe_async::maybe_async]` macro like the other functions